requirements/edx/base.txt

```diff
@@ -90,10 +92,11 @@ django-memcached-hashring==0.1.2
 python-openid==2.2.5
 python-dateutil==2.1
 
-# This module gets monkey-patched in third_party_auth.py to fix a Django 1.8 incompatibility.
-# When this dependency gets upgraded, the monkey patch should be removed, if possible.
-# We can also remove the fix to auth_url in third_party_auth/saml.py when that fix is included upstream.
-python-social-auth==0.2.12
+# we have to install social-auth-app-django from fork to get over missing migration
+# issue https://github.com/python-social-auth/social-app-django/issues/28
+# once migrations are run we can switch back to base package(social-auth-app-django==1.0.1)
+-e git+https://github.com/ziafazal/social-app-django.git@8f4688076eab6d9b6fb71bb045e779b6c627835f#egg=social-auth-app-django
```

Comment:
The recommended upgrade route from https://github.com/omab/python-social-auth/blob/master/MIGRATING_TO_SOCIAL.md is to first go to 0.2.21. I am also leery of relying on an individual developer's repo. I'd much rather us fork to edx. That being said, it's not clear this is needed.